cypress/e2e/settings.spec.cy.js

```diff
@@ -0,0 +1,67 @@
+import { checkNoErrorOnThePage,
+         checkSuccessAlert,
+         fillInTheSettingsConfigForm,
+         buttonOnTabClick,
+         buttonClick,
+         setup } from "../support/utils";
+
+describe("Settings section", () => {
+  before(() => {
+    setup();
+  });
+
+  it("Menu: Settings. Tab: Config. Validate that we can set and save credits values.", () => {
+    cy.visit("/");
+
+    // We have a bug with Tasks menu were we get exception below
+    // IndexError: index 0 is out of bounds for axis 0 with size 0
+    // Ignoring this for now as this is a default page we land on
+    // when clicking on Settings
+    cy.wait(5000);
+
+    cy.get("span", {timeout: 20000})
+      .contains("Settings")
+      .should("be.visible")
+      .click();
+
+    cy.wait(5000);
+
+    cy.get('button[role="tab"]', {timeout: 20000})
+      .should("exist")
+      .contains("Config")
+      .should("exist")
+      .click()
+
+    fillInTheSettingsConfigForm(10.00, 20.00, 30.00);
```

Comment:
Should there be a check to verify that these properties are now reflected on the page after save?

Reply:
yes, there should be. 
These are sanity tests that validate there are no exceptions or errors on the pages during object creation/deletion.
Next set of tests are (a) Correctness checks. (b) Error conditions

Reply:
> These are sanity tests that validate there are no exceptions or errors on the pages during object creation/deletion.

Gotcha. Sorry, the intended scope of the test was not apparent to me. Can you make sure there is an issue opened for that work (so we know when it's coming)? Thanks!